packages/postgres-database/src/simcore_postgres_database/utils_groups_extra_properties.py

```diff
@@ -95,34 +92,55 @@ def _merge_extra_properties_booleans(
 
 @dataclass(frozen=True, slots=True, kw_only=True)
 class GroupExtraPropertiesRepo:
+    @staticmethod
+    def _get_stmt(gid: int, product_name: str):
+        return sa.select(groups_extra_properties).where(
+            (groups_extra_properties.c.group_id == gid)
+            & (groups_extra_properties.c.product_name == product_name)
+        )
+
     @staticmethod
     async def get(
         connection: SAConnection, *, gid: int, product_name: str
     ) -> GroupExtraProperties:
-        get_stmt = sa.select(groups_extra_properties).where(
-            (groups_extra_properties.c.group_id == gid)
-            & (groups_extra_properties.c.product_name == product_name)
+        warnings.warn(
+            _WARNING_FMSG.format("get", "get_v2"),
+            DeprecationWarning,
+            stacklevel=1,
         )
-        result = await connection.execute(get_stmt)
+
+        query = GroupExtraPropertiesRepo._get_stmt(gid, product_name)
+        result = await connection.execute(query)
         assert result  # nosec
         if row := await result.first():
-            return GroupExtraProperties.from_row(row)
+            return GroupExtraProperties.from_row_proxy(row)
         msg = f"Properties for group {gid} not found"
         raise GroupExtraPropertiesNotFoundError(msg)
 
     @staticmethod
-    async def get_aggregated_properties_for_user(
-        connection: SAConnection,
+    async def get_v2(
```

Comment:
what is v2 for? is that asyncpg? I would prefer something explicit. or use the `@overload` option. see `limited_gather` for an example.

Reply:
this is suppose to dissapear in a few weeks ... so I would not spend more time here (at least in this PR which got ab it out of hand)